mods/ra/maps/allies-06a/map.yaml

```diff
@@ -0,0 +1,650 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: 06a: Iron Curtain research center
+# Infiltrate and destroy Iron Curtain research center
```

Comment:
I think these titles are too long:
### Allied:
◾Infiltrate and destroy Iron Curtain research Center

◾Protect the Chronosphere research center

◾Secure the river for the navy

### Soviet:
◾Capture the control centers